pkg/api/api.go

```diff
@@ -391,6 +391,29 @@ func (s *server) newTracingHandler(spanName string) func(h http.Handler) http.Ha
 	}
 }
 
+func (s *server) contentLengthMetricMiddleware(method string) func(h http.Handler) http.Handler {
```

Comment:
it is not needed to provide the `method`, since you can always get the `request.Method` from the actual request object inside the middleware